scripts/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "compilerOptions": {
+    "noEmit": true,
+    "strict": true,
+    "esModuleInterop": true,
+    "resolveJsonModule": true,
+    "noImplicitAny": false,
+    "experimentalDecorators": true,
+    "target": "es2019",
+    "module": "commonjs",
```

Comment:
I think you can remove this property or at least you could use a more recent type of module (e.g. es6).

Reply:
It might be the default, but we have it defined in all our tsconfig files. Also, es6 is kinda old (2015), we currently use es2019